### PR TITLE
Issue #267 - Added tooltip for option icons.

### LINKED ISSUE #267: [Forms] Add tool tips for option icons
The new UI could be seen here https://www.figma.com/file/61NHR7fwmV1Dsv4Pb6F8T7/code4ro_mv_user-management_2018.04.21-(Copy)?node-id=1120%3A31

Please add tool tip to the two icons that can show up for answers:
- isFreeText - text should be `Mark as free text` / `Mark as regular`
- isFlagged - text should be `Flag option` / `Unflag option`

create form - edit form - tooltip if needed

### PR DESCRIPTION
**Pull request by ajeurkar**
Signed-off-by: Ajinkya Jeurkar <ajinkyajeurkar@gmail.com>

### What does it fix?
Closes #267 

### How has it been tested?

- Tested locally

Demo -
Peek 2020-10-08 00-38

Screenshots -
Screenshot from 2020-10-08 00-39-06
Screenshot from 2020-10-08 00-39-13
Screenshot from 2020-10-08 00-39-18
Screenshot from 2020-10-08 00-39-23

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -6,3 +6,13 @@
 .cursor-pointer {
   cursor: pointer;
 }
+
+.custom-tooltip {
+  .tooltip-inner {
+    background-color: white;
+    color: black;
+  }
```
Comment (Utwo) [inline src/styles.scss:14]: Can you replace this block with the following bootstrap classes? 
`class="bg-white text-dark"`

More info here: https://getbootstrap.com/docs/4.5/utilities/colors/

**Review by Utwo — CHANGES_REQUESTED** [review]

File: src/styles.scss (hunk 1 above)
Comment (ajeurkar) [reply src/styles.scss:14]: @Utwo That can be done by setting the `encapsulation: ViewEncapsulation.None` for `app-icon-toggle-input` component. Let me know your inputs.

Comment (ajeurkar) [reply src/styles.scss:14]: @Utwo  We need to override the `.tooltip-inner` class of [**NgbTooltip**](https://ng-bootstrap.github.io/#/components/tooltip/api) So, we cannot directly apply the boostrap classes. Let me know your inputs.

Comment (Utwo) [reply src/styles.scss:14]: You are right! Thanks!

**Review by Utwo — APPROVED** [review]

**Review by aniri — APPROVED** [review]

**Pull request merged**